zulip/integrations/bridge_with_matrix/README.md

```diff
@@ -1,20 +1,21 @@
 # Matrix <--> Zulip bridge
 
-This acts as a bridge between Matrix and Zulip. It also enables a
-Zulip topic to be federated between two Zulip servers.
+This acts as a bridge between Matrix and Zulip.
 
-## Usage
+### Enhanced Features
+- Supporting multiple (Zulip topic, Matrix channel)-pairs.
+- Handling files according to their mimetype.
 
-### For IRC bridges
 
-Matrix has been bridged to the listed
-[IRC networks](https://github.com/matrix-org/matrix-appservice-irc/wiki/Bridged-IRC-networks),
-where the 'Room alias format' refers to the `room_id` for the corresponding IRC channel.
+## Installation
 
-For example, for the freenode channel `#zulip-test`, the `room_id` would be
-`#freenode_#zulip-test:matrix.org`.
+Run `pip install -r requirements.txt` in order to install the requirements.
+
+In case you'd like encryption to work, you need pip to install the `matrix-nio`
+package with e2e support: `matrix-nio[e2e]`. Caution: you have to install the
```

Comment:
Some people who are not familiar with Python package installation might not be aware that you have to install it via `pip install matrix-nio[e2e]`.

Reply:
done, thanks :)